filters/exclusions.txt

```diff
@@ -257,5 +257,10 @@ _campaign=$popup
 ||github.io^$third-party,xmlhttprequest,domain=~facebook.com|~gametactic.org|~github.com|~github.io|~tumblr.com|~wottactic.com
 !###########################
 !
+!### NoCoin Filter List ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/84173
+/.*##\.ad$/
```

Comment:
Checked again. The rule in Polish filter is limited to PL only - `***,pl##.ad`
But hufilter really has wide rule `##.ad`. So better revert to `/^##\.ad$/`

Reply:
Done.